### PR TITLE
Improve portforward command's description 

### PR DESCRIPTION
**Pull request by WanLinghao**
> /kind documentation

**What this PR does / why we need it**:
When execute kubectl port-forward command, it may trigger error when node has no 'socat' command installed, it looks like:
`an error occurred forwarding 80 -> 80: error forwarding port 80 to pod def05a827d0b2a68586e183ac2ce8e77cdc1afc7bca0c5956fe75a1f5e0651c8, uid : unable to do port forwarding: socat not found.`

This patch adds a hint about this.

```release-note
NONE
```

### CODE REVIEW

File: pkg/kubectl/cmd/portforward/portforward.go
Comment (neolit123) [inline pkg/kubectl/cmd/portforward/portforward.go]: please change the message to:
`This command requires the node to have 'socat' installed.`

**Review by neolit123 — COMMENTED** [review]
/priority backlog
/kind cleanup

**Review by neolit123 — COMMENTED** [review]
/lgtm
thanks for the update.
at this point this needs approval from the kubectl maintainers.

**Comment by WanLinghao** [comment]
@juanvallejo PTAL

**Comment by juanvallejo** [comment]
Thanks

/lgtm

cc @soltysh

**Review by soltysh — APPROVED** [review]
/approve

**Pull request merged**
